public/app/features/explore/Wrapper.tsx

```diff
@@ -31,7 +31,7 @@ export class Wrapper extends Component<WrapperProps> {
 
     return (
       <div className="page-scrollbar-wrapper">
-        <CustomScrollbar autoHeightMin={'100%'} autoHeightMax={''} className="custom-scrollbar--page">
+        <CustomScrollbar autoHeightMin={'100%'} autoHeightMax={''} page={true}>
```

Comment:
Nice catch, but do you think page a suitable name for this property? I think it seems a bit too generic?